CONTRIBUTING.md

```diff
@@ -4,6 +4,27 @@ Contributing to Skydive
 Your contributions are more than welcome. Please read the following notes to
 know about the process.
 
+Making Changes:
+---------------
+
+Before pushing your changes, please make sure the tests and the linters pass:
+
+* make fmt
+* make test
+* make test.functionals
```

Comment:
No need to export SKYDIVE_ANALYZERS when using the 'standalone mode'.

make test.functionals VERBOSE=true TIMEOUT=10m ARGS='-standalone'